docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -1,53 +1,163 @@
 ---
 slug: pricing-and-billing
-description: >-
-  Pricing and billing for Semgrep App
+append_help_link: true
+title: Pricing and billing
+description: "Semgrep CLI and CI are free to use. Semgrep App has both free and paid tiers, each with their own features and levels of support."
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
+
 # Pricing and billing
 
-## Can I use Semgrep for free?
+The following Semgrep tools are free to use:
+
+* Semgrep CLI
+* Semgrep CI
+* Semgrep App (through the Community tier)
+
+Semgrep CLI is free open-source software (FOSS), licensed under LGPL 2.1. Semgrep CI is source-available. All features for both Semgrep CLI and Semgrep CI are free.
+
+Semgrep App, which builds on Semgrep CLI and CI, enables developer teams to create their own institutional code standards, enforce them, and analyze findings. Semgrep App has both free and paid tiers.
+
+## Choosing a Semgrep App tier
+
+Semgrep App has three tiers:
+
+| Tier | Price | Description |
+| ---- | ------- | -------- | 
+| **Community** | Free | For general-purpose security scanning.  |
+| **Team** | $40/mo per seat | For the enforcement of company-specific coding standards, private rules, and the analysis of findings.  |
+| **Enterprise** | Custom pricing | For custom solutions, including deployment into virtual private clouds (VPCs) with dedicated technical support.  |
+
+Semgrep App integrates with a GitHub or GitLab account at either the organizational or individual level. There is no limit to the number of members in an organization for all tiers.
+
+Billing depends on the number of **seats**. Any member whose code has been scanned by Semgrep App takes up a seat.
+
+Members who do **not** run any scans on their code do not need a seat and are not billed. Semgrep Team tier features apply to the entire organization, even if not all members are billed for a seat.
+
+### Comparing free and paid tier features
+
+#### Organizational management
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- |
+| **Member limit** | ♾️ unlimited | ♾️  unlimited | ♾️  unlimited |
+| **Project limit** |♾️  unlimited | ♾️  unlimited | ♾️  unlimited |
+
+#### Findings, language support, and rules
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- | 
+| **Findings retention** | 1 month | 1 year | 1 year+ (custom) |
+| **Community rule registry** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Named rules** | ✔️ yes | ✔️ yes | ✔️  yes |
+| **Private rules** | ❌ no | ✔️ yes | ✔️  yes |
+| **Custom rule messages** | ❌ no | ✔️  yes | ✔️  yes |
+| **Custom language support** | ❌ no | ❌ no | ✔️  yes |
+
+Notes:
+
+* **Named rules** are rules that have been saved through the [Playground](https://semgrep.dev/editor). Named rules  created by Community tier members are public and available in the Playground.
```

Comment:
We can say in one sentence what is Playground, so people do not have to open the link.

Reply:
Will do!